src/prodbg/main/src/windows.rs

```diff
@@ -150,6 +175,7 @@ impl Windows {
                   sessions: &mut Sessions,
                   view_plugins: &mut ViewPlugins,
                   backend_plugins: &mut BackendPlugins) {
+        sleep(Duration::from_millis(100));
```

Comment:
It is part of debug code as mentioned earlier. Without FPS restriction
prodbg eats one core just for to update interface.

2016-07-04 17:40 GMT+08:00 Daniel Collin <notifications@github.com>:

> In src/prodbg/main/src/windows.rs
> <https://github.com/emoon/ProDBG/pull/218#discussion_r69432981>:
>
> > @@ -150,6 +175,7 @@ impl Windows {
> >                    sessions: &mut Sessions,
> >                    view_plugins: &mut ViewPlugins,
> >                    backend_plugins: &mut BackendPlugins) {
> > +        sleep(Duration::from_millis(100));
>
> Why is this here? It make the UI quite laggy.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/emoon/ProDBG/pull/218/files/d4db4aed4c9f97d6991801a031c077eb57d239f0#r69432981>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AANveHbP3VQHOxJESZeJn0AoGMDx7m5tks5qSNUmgaJpZM4JELvA>
> .
>